MediaBrowser.Model/Lyrics/LyricLineCue.cs

```diff
@@ -0,0 +1,42 @@
+namespace MediaBrowser.Model.Lyrics;
+
+/// <summary>
+/// LyricLineCue model, holds information about the timing of words within a LyricLine.
+/// </summary>
+public class LyricLineCue
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LyricLineCue"/> class.
+    /// </summary>
+    /// <param name="position">The start of the character index of the lyric.</param>
+    /// <param name="width">The count of the characters in the lyric.</param>
+    /// <param name="start">The start of the timestamp the lyric is synced to in ticks.</param>
+    /// <param name="end">The end of the timestamp the lyric is synced to in ticks.</param>
+    public LyricLineCue(int position, int width, long start, long end)
```

Comment:
I think adding both width and end may be a bit redundant, let's choose one and stick with that. I'm not sure if we have other places in the API that we use as reference.

Reply:
I've decided to add "end" because it's more clear on exactly where the end point is, and is consistent with "start". However, if we need to use width anyway, the code for it is in the repo history anyway